salt/metalk8s/container-engine/containerd/running.sls

```diff
@@ -0,0 +1,21 @@
+# NOTE: This state just ensure containerd running and "ready"
+# To restart containerd you need to use a `watch_in` in another state 
+# that include this one
+# ```
+# - watch_in:
+#   - service: Ensure containerd running
+# ```
+
+Ensure containerd running:
+  service.running:
+    - name: containerd
+    - enable: True
+    - init_delay: 2
+
+Ensure containerd is ready:
+  test.configurable_test_state:
```

Comment:
Any reason `cri.ready` is not implemented as some module?

Reply:
Would need to create a state module in addition to wrap this call. Using slots looks simpler to me :)

Reply:
WDYM ? As a salt state module ?